azure-pipelines/templates/steps/auth-client/docker-tasks.yml

```diff
@@ -10,7 +10,7 @@ steps:
     echo =============================================
     echo Run unit and instrumented inside docker container
     echo =============================================
-    docker run --privileged --cpus="3" --memory="12g" -v "$PWD":/home/gradle/ -w /home/gradle/ authclient.azurecr.io/samples/dbi-instrumented-api30 sh ../../scripts/run-instrumented-tests.sh
+    docker run --privileged --cpus="3" --memory="12g" -v "$PWD":/home/gradle/ -w /home/gradle/ authclient.azurecr.io/samples/dbi-instrumented-api30 sh scripts/run-instrumented-tests.sh
```

Comment:
Should this be an absolute path?

Reply:
for some reason, it cant 
![image](https://user-images.githubusercontent.com/76129899/116174700-cc383000-a6c3-11eb-8a1a-b221318b839d.png)

Reply:
Wild.